### PR TITLE
feature: toggle component for widget

### PR DESCRIPTION
**Pull request by arpitnath**
**Fixes** : #498 

** Toggle button as a widget**
Screenshot 2021-08-12 at 4 30 58 PM

Screenshot 2021-08-12 at 2 37 10 PM

**Description** 
component: ToggleSwitch
events: onChange
properties: label
default widget size: width: 150px, height: 50px

**Loom** : https://www.loom.com/share/4fe9e5a5b86e4516a9858e7dba6cf361

### CODE REVIEW

File: frontend/src/Editor/Components/Toggle.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    
+  const [on, setOn] = React.useState(false)
+  const textColorProperty = component.definition.styles.textColor;
+  const textColor = textColorProperty ? textColorProperty.value : '#000';
+
+  function toggleValue(e) {
+    const toggled = e.target.checked;
+    onComponentOptionChanged(component, 'value', toggled);
+    if (toggled) {
+      onEvent('onToggle', { component });
```
Comment (Navaneeth-pk) [inline frontend/src/Editor/Components/Toggle.jsx:43]: Can we have just `onChange` event instead of onToggle and onUnToggle?

Comment (arpitnath) [reply frontend/src/Editor/Components/Toggle.jsx:43]: yes I guess.

File: frontend/src/Editor/Components/components.js [hunk 2]
```diff
@@ -365,6 +365,55 @@ export const componentTypes = [
       }
     }
   },
+  {
+    name: 'Toggle',
+    displayName: 'Toggle',
```
Comment (Navaneeth-pk) [inline frontend/src/Editor/Components/components.js:370]: Please change to `Toggle switch`

File: frontend/src/Editor/Components/Toggle.jsx (hunk 1 above)
Comment (Navaneeth-pk) [reply frontend/src/Editor/Components/Toggle.jsx:43]: coo, do it please

**Comment by Navaneeth-pk** [comment]
@arpitnath the label is not visible in the video 🤔

**Comment by arpitnath** [comment]
updated the screenshots and loom recording

**Comment by Navaneeth-pk** [comment]
@arpitnath label and the toggle should be aligned.

**Comment by arpitnath** [comment]
Screenshot 2021-08-13 at 11 11 06 AM

**Update**:  Changes done

**Pull request merged**